src/main/java/org/damap/base/rest/DmpDocumentResource.java

```diff
@@ -41,34 +43,74 @@ public class DmpDocumentResource {
    */
   @GET
   @Path("/{dmpId}")
+  @Deprecated(since = "4.3.0")
   public Response exportTemplate(
       @PathParam("dmpId") long dmpId, @QueryParam("template") ETemplateType template) {
     log.info("Return DMP document file for DMP with id=" + dmpId);
 
+    return this.export(dmpId, template, true, "docx");
+  }
+
+  @GET
+  @Path("/{dmpId}/export")
+  @Produces(MediaType.APPLICATION_OCTET_STREAM)
+  public Response export(
+      @PathParam("dmpId") long dmpId,
+      @QueryParam("template") ETemplateType template,
+      @QueryParam("download") Boolean download,
+      @QueryParam("filetype") String filetype) {
+
+    if (download == null) {
+      download = true;
+    }
+
+    if (filetype == null || (!filetype.equals("pdf") && !filetype.equals("docx"))) {
+      filetype = "docx";
+    }
```

Comment:
major: this should be handled on the service level as there we should know how to deal with different values.

Reply:
Done